Taskfile.yml

```diff
@@ -51,25 +49,29 @@ tasks:
 
   compile:fe:
     desc: Runs less, pug and node
-    dir: public
+    dir: web
     sources:
-      - css/semaphore.less
-      - html/*.pug
-      - html/**/*.pug
+      - resources/css/semaphore.less
+      - resources/html/*.pug
+      - resources/html/**/*.pug
     generates:
-      - css/semaphore.css
-      - html/*.html
-      - html/**/*.html
-      - js/bundle.json
-    cmds:
-      - lessc css/semaphore.less > css/semaphore.css
-      - pug $(find ./html/ -name "*.pug")
-      - node ./bundler.js
+      - public/css/semaphore.css
+      - public/html/*.html
+      - public/html/**/*.html
+      - bundle.json
+    cmds:
+      - mkdir -p public/css
+      - mkdir -p public/html
+      - mkdir -p public/fonts
```

Comment:
can we keep these directories in the repo with a .gitkeep so we don't need to try and make them every time (as they will usually exist)

Reply:
Fixed